git_machete/tests/functional/test_machete.py

```diff
@@ -12,65 +12,65 @@
 
 class SandboxSetup:
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.file_dir = os.path.dirname(os.path.abspath(__file__))
         self.remote_path = os.popen('mktemp -d').read().strip()
         self.sandbox_path = os.popen('mktemp -d').read().strip()
 
     @staticmethod
-    def execute(command):
+    def execute(command: str) -> None:
         result = os.system(command)
-        assert result == 0, "%s returned %i" % (command, result)
+        assert result == 0, f"{command} returned {result:d}"
 
-    def new_repo(self, *args):
+    def new_repo(self, *args: str) -> 'SandboxSetup':
         os.chdir(args[0])
         if len(args) > 1:
             opt = args[1]
-            self.execute('git init %s' % opt)
+            self.execute(f'git init {opt}')
         else:
             self.execute('git init')
         return self
 
-    def new_branch(self, branch_name):
-        self.execute('git checkout -q -b %s' % branch_name)
+    def new_branch(self, branch_name: str) -> 'SandboxSetup':
```

Comment:
Whoops so for class return types... quotes should used? 😯 as in, you can't write `-> SandboxSetup`?

Reply:
yup

Reply:
https://stackoverflow.com/questions/15853469/putting-current-class-as-return-type-annotation

Reply:
can be resolved with tihs `from __future__ import annotations` too

Reply:
Ok, just asking out of curiosity, leave as is